doc/source/whatsnew/v0.19.0.txt

```diff
@@ -43,7 +43,7 @@ Backwards incompatible API changes
 .. _whatsnew_0190.api:
 
 
-
+- Concating multiple objects will no longer result in automatically upcast to `float64`, and instead try to find the smallest `dtype` that would suffice (:issue:`13247`)
```

Comment:
Yes, because was tagged with "Next major release", so I thought it was supposed to go in 0.19.  https://github.com/pydata/pandas/issues/13247.  I can easily move back.